lib/components/user/places/favorite-place-screen.js

```diff
@@ -38,8 +40,8 @@ const Container = styled.div`
 
 // The form fields to validate.
 const validationSchemaShape = {
-  address: yup.string().required('Please set a location for this place'),
-  name: yup.string().required('Please enter a name for this place')
+  address: yup.string().required('components.FavoritePlaceScreen.invalidAddress'),
+  name: yup.string().required('components.FavoritePlaceScreen.invalidName')
```

Comment:
If using a validation error message to determine the message ID, do not use a message ID string to avoid confusion.